2-projects/1-todo-list/script.js

```diff
@@ -17,9 +48,47 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+
+  event = document.getElementById("todoInput");
+  let userOutput = event.value;
+  if (!userOutput) {
+    alert("Please enter a task.");
+  } else {
+    let list = document.getElementById("todo-list");
+    let li = document.createElement("li");
+    li.classList.add(
+      "list-group-item",
+      "d-flex",
+      "justify-content-between",
+      "align-items-center"
+    );
+    li.innerHTML = userOutput;
+    list.appendChild(li);
+    let span = document.createElement("span");
+    span.classList.add("badge", "bg-primary", "rounded-pill");
+    li.appendChild(span);
+    let i1 = document.createElement("i");
+    i1.className = "fa fa-check";
+    i1.ariaHidden = "true";
+    let i2 = document.createElement("i");
+    i2.className = "fa fa-trash";
+    i2.ariaHidden = "true";
+    span.append(i1, i2);
+    event.value = ""; // clear input field after button press
+    document.getElementsByClassName("fa-check");
+    i1.addEventListener("click", () => {
+      li.classList.toggle("lineThrough");
```

Comment:
Also great use of the toggle feature. I did it a different way, but this is the best way to handle this kind of scenarios.